alphafold3_pytorch/inputs.py

```diff
@@ -168,6 +168,7 @@ class AtomInput:
     pde_labels:                 Int['n n'] | None = None
     plddt_labels:               Int[' n'] | None = None
     resolved_labels:            Int[' n'] | None = None
+    chains:                     Tuple[int | None, int | None] = (None, None)
```

Comment:
a type of `Int[' 2']` tensor at the `AtomInput` would lead to less issues, and `Int['b 2']` on `BatchAtomInput`

Reply:
for `None`, can just use something out of range, like say `-1`

Reply:
Sg

Reply:
Ah, also got to update the forward()

Reply:
And I also need to listen to myself and make `chains` optional :)